Taskfile.yaml

```diff
@@ -38,11 +38,16 @@ tasks:
     cmds:
       - task: lint:chart:redpanda
 
-  lint:chart:redpanda:
+  chart:redpanda:lint:
     cmds:
       - ct lint --config .github/ct-redpanda.yaml --github-groups
       - .github/check-ci-files.sh charts/redpanda/ci
 
+  chart:redpanda:generate:
+    cmds:
+      - go run ./cmd/genpartial/main.go -out charts/redpanda/values_partial.gen.go -struct Values ./charts/redpanda
+      - go run ./cmd/genvalues/main.go > charts/redpanda/values.schema.json
```

Comment:
going the reverse of what we had, interesting.

Reply:
How do you mean? Were we generating something from the json schema previously?

Reply:
exactly. I took an existing tool to take schemas from us and create apis.

Reply:
What was the output of that and was it ever automated anywhere? The `values.schema.json` file as it currently exists is missing a good chunk of fields and it's fairly malformed as well.

Reply:
No, we never got around to make it automated, ill send you the link so you can see the repo.

Reply:
https://github.com/alejandroEsc/generate